content/response-to-reviews/therapeutics-v1.md

```diff
@@ -0,0 +1,87 @@
+The detailed critiques have been moved to the appendix, and the narratives surrounding failed approaches has been streamlined so as not to over-emphasize them.
+**
+
+It also seems dangerous to infer researcher biases like p-hacking without more definitive data that support these potentially damaging claims.
+
+**
+The sentence in question has been modified to read: "Additionally, disparities between the pre-registered and published protocols raises concerns about experimental design."
+**
+
+1.3 This article as presented seems premature. I was especially confused by the statement in the abstract that "this review will be updated as progress is made." Is that even possible at mSystems? 
+
+**
+The project is built on Manubot, a relatively new resource for collaborative writing that allow for version control and continual editing.
+An up-to-date version of the manuscript is available online at all times, but _mSystems_ has agreed to review a version of record and to allow the submission of updates as more evidence becomes available over time, in the style of a living review.
+**
+
+More importantly, the conclusion for all but 1-2 therapeutics is that more data is needed, leaving me wondering what the value of this review is at time present. One useful contribution would be to rank the promise of each approach. Each has issues, but which ones are worth investing more resources in?
+
+**
+We hope that the recontextualization of the content serves to address the concerns related to the usefulness of the project -- the goal is to organize the available evidence to allow scientists to get or stay up to speed on the status of these therapeutics without needing to dive independently into the exceptionally large body of literature.
+**
+
+1.4 Figure 1 - I'm not sure this adds much to the current topic.
+
+**
+Figure 1 has been modified to show the relative trajectories of the SARS-CoV-1 and SARS-CoV-2 viral outbreaks (data were not available for MERS).
+The goal of this figure is to show why pharmacology plays such a bigger role in the present pandemic than it has in the past, as well as to emphasize how much more feasible clinical trials are based on the differences in caseload.
+We hope the relevance is improved with these changes.
+**
+
+1.5 Figure 2 - panel labels (letters) are needed. 
+
+**
+This change has been made.
+**
+
+The y-axis of panel 4 is difficult to read - the panel should either be stretched or some of the less common approaches could be trimmed off. The "other" category could also be removed.
+
+**
+These changes have been made.
+**
+
+More importantly, I worry that this figure is immediately out of date, is there any indication that these trends will be stable over time? If not, why is it important to know what fraction of trials are in Phase 2 right now?
+
+**
+As part of this project, we developed software to pull data from the web and update these figures every night.
```

Comment:
Should we add the link to https://greenelab.github.io/covid19-review/ in the figure caption and say something like
> See <https://greenelab.github.io/covid19-review/> for the most recent version of this figure.

Then the reviewer may better understand why we include it and the benefits of the automatic updates.

Reply:
Sounds like a great idea to me!
```suggestion
As part of this project, we developed software to pull data from the web and update these figures every night.
See https://greenelab.github.io/covid19-review/ for the most recent version of this figure.
```